config/webhook/manifests.yaml

```diff
@@ -38,60 +38,60 @@ webhooks:
     service:
       name: webhook-service
       namespace: system
-      path: /validate-hnc-x-k8s-io-v1alpha2-subnamespaceanchors
```

Comment:
It seems like the order is not consistent. What might be the reason? Another version of controller-gen? If so, I would prefer to re-generate this is in a separate PR.

Reply:
This reordering was caused by adding functionality to the `objects` validator. By moving the common validation logic elsewhere (e.g. `forest`), this will revert to the old order.